package/kernel/linux/modules/netsupport.mk

```diff
@@ -922,6 +922,22 @@ endef
 $(eval $(call KernelPackage,sched-ipset))
 
 
+define KernelPackage/sched-fq-pie
+  SUBMENU:=$(NETWORK_SUPPORT_MENU)
+  TITLE:=Flow Queue w Proportional Integral Enhanced (FQ-PIE)
```

Comment:
Is the `w` there intentionally?
The upstream kernel names it like this: `Flow Queue Proportional Integral controller Enhanced (FQ-PIE)`

Reply:
Had to squeeze out a few chars to get it to appear in the menu, there is a max limit.

Reply:
I would prefer `Flow Queue Proportional Integral Enhanced (FQ-PIE)`

Reply:
modded